client/src/sections/resources/PowerfulGlove.tsx

```diff
@@ -27,7 +27,8 @@ const PowerfulGlove = () => {
     <Tile
       header="Powerful Glove"
       imageUrl="/images/itemimages/Pglove.gif"
-      itemToEquip={$item`Powerful Glove`}
+      linkedContent={$item`Powerful Glove`}
```

Comment:
wanted to just say i love this format of "linkedContent", think it is a great phrase for this; great job beldur. :-)